internal/store/store.go

```diff
@@ -288,7 +274,7 @@ type BasicConfig interface {
 func PublicAuthenticatedPreset(c AuthenticatedConfig) Option {
```

Comment:
I think now `PublicAuthenticatedPreset` and `PrivateAuthenticatedPreset` should work the same right? should we go to just one?

Reply:
same for `PublicUnauthenticatedPreset` and `PrivateUnauthenticatedPreset`